docs/cubeviz/displaycubes.rst

```diff
@@ -45,18 +45,50 @@ visible (checked) or invisible (unchecked).
 Changing Wavelength Slice
 =========================
 
-To change the wavelength slice, click and drag the :guilabel:`slider` at the top of the Cubeviz UI
+To change the wavelength slice, click and drag the :guilabel:`slider` at the top of the Cubeviz User Interface (UI)
 to the left or right in order to get a lower or higher slice number, respectively. Another option is to
 click the :guilabel:`circle` that controls the location on the :guilabel:`slider` and use the
 :guilabel:`left` or :guilabel:`right` arrow keys on your keyboard to find a slice number with more precision.
 A third option is to type a slice number into the :guilabel:`text box` to the right of the :guilabel:`slider`
 and press :guilabel:`enter` on your keyboard. A fourth option is to use the :guilabel:`up/down arrows` inside
 the :guilabel:`text box` to raise/lower the slice number.
 
-Defining and Selecting Region Subsets
-=====================================
+Defining Spatial Regions
+========================
+
+Regions can be created by first clicking the :guilabel:`hammer and screwdriver` icon and
+selecting either the :guilabel:`rectangular region of interest` icon or the
+:guilabel:`circular region of interest` icon, then clicking and dragging (using the mouse) until the
+desired region is covered by a colored shape, after which you release the mouse button.
+Once you have completed the action of clicking and dragging, there is an opportunity to reposition
+the region of interest by clicking on it and moving the region to a different location in the image viewer.
```

Comment:
```suggestion
the region of interest by dragging the region to a different location in the image viewer.
```

I read this as a full KeyOn and KeyOff event, followed by dragging the mouse with no buttons pressed. I thought it was a click-and-drag event in order to move it? If I'm mistaken, feel free to ignore this one!

Reply:
I'm not exactly sure what the question is. You click and drag to create the region, which is completed once the mouse button is released. The user then has an opportunity to reposition the region by clicking on it once again and moving (dragging) the region to a different location. Once the user clicks off of the region, it is locked.

Reply:
I think I understand now, I'll switch the wording to "clicking and dragging the region"